### ISSUE TITLE
Update npm package to main

### ISSUE THREAD

**Issue by NilsBaumgartner1994** [zamplyy/react-native-nice-avatar#5]
The npm package contains a different source code than the git repo

**Comment by zamplyy** [comment]
Hi! Thanks for your issue, 
What do you mean by this, the npm package should contain the built files right? 
Or what other inconsistencies do you find?

**Comment by NilsBaumgartner1994** [comment]
The npm package does not have the types file with sex, .. and still has the upper case values:

NPM Package Content:
...
```
  noseType: 'Round' | 'Pointed' | 'Curved';
```


Git Repo Content:
...
```
export type NoseStyle = 'short' | 'long' | 'round';
```

**Comment by NilsBaumgartner1994** [comment]
Oh wait, there is also https://www.npmjs.com/package/react-native-nice-avatar a same named package

**Comment by zamplyy** [comment]
Sorry for the misunderstanding, i first published to github packages before deciding on npm.

**Comment by NilsBaumgartner1994** [comment]
Yeah no problem :-) I like your package very much and it is something that is needed !
Would be nice to see the smaller bugs to be solved like #11 

I want to use it as an alternative to https://getavataaars.com/
